rule/struct_tag.go

```diff
@@ -322,6 +328,28 @@ func (w lintStructTagRule) checkYAMLTag(options []string) (string, bool) {
 	return "", true
 }
 
+func (w lintStructTagRule) checkURLTag(options []string) (string, bool) {
+	var delimiter = ""
+	for _, opt := range options {
+		switch opt {
+		case "int", "omitempty", "numbered", "brackets":
```

Comment:
It looks like we miss the following: `unix`, `unixmilli`, `unixnano`

Also, I'd consider adding support for `layout` and `del` struct tags, since they are a part of the spec for [the ](https://pkg.go.dev/github.com/google/go-querystring/query).

Reply:
I' ve added unix* options.

`layout` and `del` : [almost no-body uses them](https://github.com/search?q=+layout%3A%22+path%3A*.go+language%3AGo&type=code&ref=advsearch) so I think it does not worth supporting these tags.